spec/pg/connection_spec.rb

```diff
@@ -731,6 +731,31 @@
 		expect( @conn.options ).to eq( "" )
 	end
 
+	it "connects without port and then retrieves the default port" do
+		gate = Helpers::TcpGateSwitcher.new(
+				external_host: 'localhost',
+				external_port: ENV['PGPORT'].to_i,
+				internal_host: "127.0.0.1",
+				internal_port: 5432,
```

Comment:
> Should we add `DEF_PGPORT` to PG::Constants?

That constant can/should be used here, in case someone configured with some other `--with-pgport`. On the other hand, who does that?!

Reply:
OK, done.